tests/unittests/tests-base64/tests-base64.c

```diff
@@ -303,13 +303,13 @@ static void test_base64_07_stream_decode(void)
 
 static void test_base64_08_encode_16_bytes(void)
 {
-    const int buffer_size = 16;
+    enum { buffer_size = 16 };
     unsigned char buffer[buffer_size];
```

Comment:
Why these are initialized as they are is a little bit strange (one could just use a macro for that), but let's fix this in a separate PR.